app/controllers/api/v1/users_controller.rb

```diff
@@ -1,57 +1,28 @@
 # frozen_string_literal: true
 
 class Api::V1::UsersController < Api::V1::BaseController
-  skip_before_action :authenticate_user!, only: [:create]
-  skip_before_action :authenticate_user_using_x_auth_token, only: [:create]
+  skip_before_action :authenticate_user!, only: :create
+  skip_before_action :authenticate_user_using_x_auth_token, only: :create
 
-  before_action :set_user, only: [:show, :edit, :update, :destroy]
+  before_action :set_user!, only: %i[show destroy]
 
   def show
-    if @user
-      render json: @user
-    else
-      respond_with_error "User with id #{params[:id]} not found.", :not_found
-    end
+    render status: :ok, json: @user
```

Comment:
Use respond_with_success. Make changes wherever necessary.

Reply:
^ Not sure why above conversation was resolved even when changes weren't made yet. https://github.com/bigbinary/granite/blob/master/app/controllers/concerns/api_responders.rb#L18-L20